zoo-css-challenge/style.css

```diff
@@ -9,6 +10,13 @@ header {
   padding: 20px;
 }
 
+#logo {
+  border-width: 0;
+}
```

Comment:
```suggestion
```
Alternatively, we can delete the `img` selector below on line 40 which adds the border

```diff
-img {
-  border: 5px solid #8075ff;
-}
```

Reply:
Ahh yes! Thanks for the tips and for reviewing my work.